src/Components/MovieCard/MovieCard.css

```diff
@@ -5,28 +5,35 @@
   flex-direction: column;
   align-items: center;
   text-align: center; 
-  height: 300px; 
+  height: 325px; 
   overflow: hidden; 
   width: 30%;
   border-radius: 15px;
-  
+  background-color: #696969d6;
```

Comment:
The movie cards look great but we might have to revisit if this will pass the color contraster for accessibility.